model-registration.md

```diff
@@ -90,31 +92,35 @@ Steps to add a geographic coverage by building a bounding box:
   
 3. Select `ADD REGION TO MAP`
 
-Once you have added your geographic areas, Click `SUBMIT MODEL` to move the next step in the registration process.
+    >Note : The map does not automatically zoom in on newly added coordinates. You can zoom in manually to find your bounding box. If it is incorrect, you can remove it by clicking the X on the coordinates in the Selected Regions above.
+
+Once you have added your geographic areas, click `SUBMIT MODEL` to move the next step in the registration process.
 
 ## Containerization and Execution
 
 To launch the model execution environment, you will need to select a base image from the available drop-down menu. You may also provide the ID of an existing Debian (Ubuntu) based Docker image on Dockerhub to use as your starting point.
 
-After selecting the appropriate image you should launch your registration worker.
+![Select Worker](imgs/provision-select.png)
+
+After you've selected an image image, clicking `LAUNCH` will bring you to the `provisioning` page, where you'll wait to be redirected to the terminal once your Docker image has been loaded.
 
-![Select Worker](imgs/select-worker.png)
+![Provisioning](imgs/provisioning.png)
 
 ### Building your model
 
 You will build your model image inside the model execution environment (a Docker container). While all models are different, the general approach is to:
 
 - Clone your model code into the container.
 - Install any requirements needed to run your model.
-- Test model execution. This can be an iterative approach in that you can make several test runs to ensure that you model is running and producing the expected results.
-- Once you have a successful model run, Dojo will need to learn how you run your model, what input parameters you want to expose to the end-user, metadata about your output file(s), and the directory where your results are located. [Quick Reference](#quick-command-reference) is a short introduction to the commands you will need followed by more in depth discussions and demonstrations of each command.
+- Test model execution. This can be an iterative approach in that you can make several test runs to ensure that your model is running and producing the expected results.
+- Once you have a successful model run, Dojo will need to learn how you run your model, what input parameters you want to expose to the end-user, metadata about your output file(s), and the directory where your results are located.
 
 
 ### Dojo Commands:
-Take a look at the model registration commands [cheatsheet](./cheatsheet.md) for a reference for Dojo's containerization environment commands.
+Along with the Debian terminal commands you'll be able to execute in the virtual terminal as you configure your model, Dojo has a number of custom commands to help you register your model. You can type `dojo` at any time while in the terminal to bring up the helper text in the terminal, or view the [command cheatsheet](./cheatsheet.md).
```

Comment:
"Debian terminal commands" - bash commands? Unix commands? I'm trying to say the basic terminal commands that you can run, as we don't really mention that this is what you'll be doing anywhere else (though it is obvious, I suppose).

Reply:
Maybe `Linux terminal commands` since Debian might be too restrictive.